src/seedu/addressbook/data/person/Person.java

```diff
@@ -87,4 +90,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        final StringBuilder builder = new StringBuilder();
+        for (Printable printable : printables) {
+            builder.append(printable.getPrintableString() + '\n');
```

Comment:
Nice try with this. You may also consider formatting strings with `Formatter`, from java.util.Formatter.